### PR TITLE
Test AsciiSignalIO

### PR DESCRIPTION
**Pull request by legouee**

### CODE REVIEW

**Review by apdavison — CHANGES_REQUESTED** [review]
These tests are a good start, but they're not fully checking the behaviour. I suggest to use numpy.testing.assert_array_equal to check the full loaded array, not just the array lengths.

**Comment by JuliaSprenger** [comment]
@legouee Any updates on this?

File: neo/io/asciisignalio.py [hunk 1]
```diff
@@ -196,7 +196,13 @@ def read_segment(self, lazy=False):
                                 usecols=self.usecols,
                                 skip_header=self.skiprows,
                                 dtype='f',
+                                filling_values=None,
+                                comments='""',
+                                names=None,
+                                loose=True,
                                 invalid_raise=False)
+
+            nan = np.isnan(sig[:])
```
Comment (JuliaSprenger) [inline neo/io/asciisignalio.py:205]: This variable is not used later on. Is there more code missing here?

Comment (legouee) [reply neo/io/asciisignalio.py]: I fixed it.

**Comment by JuliaSprenger** [comment]
Thanks for adding these extensive tests. @apdavison I think @legouee adressed your requested changes. Can you confirm?
@legouee Can you fix the remaining pep8 issues?

**Comment by JuliaSprenger** [comment]
Thanks for the fixes and sorry for the delay. It's merged now!

**Pull request merged**